feat: add documentation as readme.md file for each SDK plugins #235 (#240)
AmadeusITGroup/otter: packages/@ama-sdk/core/src/plugins/fetch-cache/readme.md

## Fetch Cache

Plugin to set the cache option of the fetch API for every request it is applied to.

> For older browsers relying on a polyfill that do not support the fetch 'cache' option, you can:

Reviewer: ```suggestion
For older browsers relying on a polyfill that does not support the fetch 'cache' option, you can:
```

> - specify the content of the 'cache-control' header, that will add a 'pragma' header too in case of 'no-cache'

Reviewer: ```suggestion
- specify the content of the 'cache-control' header, which will also add a 'pragma' header in case of 'no-cache'
```
